### PR TITLE
Task #48975 [Admin] Add Template

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/48975

## WHAT (optional)
- Add Template Admin
## Evidence (Screenshot or Video)
Screenshot_5

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/controllers/admin/admin_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Admin::AdminController < ApplicationController
+  layout "admin"
```
Comment (ghost) [inline app/controllers/admin/admin_controller.rb:2]: Thêm code kiểm tra role

**Review by ghost — COMMENTED** [review]
cmt

**Comment by anhlh-3049** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**